api/src/objects/drop.rs

```diff
@@ -60,17 +60,26 @@ impl Drop {
         let now = Utc::now().naive_utc();
         let scheduled = self.drop.start_time.map(|start_time| now < start_time);
         let expired = self.drop.end_time.map(|end_time| now > end_time);
+        let paused_at = self.drop.paused_at;
+
```

Comment:
Can you add field for paused_at on the object?

Set the docs comment to "The `paused_at` field represents the date and time in UTC when the drop was paused. If it is `null`, the drop is currently not paused."